intake_solr/source.py

```diff
@@ -28,18 +30,25 @@ class SOLRSequenceSource(base.DataSource):
     zoocollection: bool or str
         If using Zookeeper to orchestrate SOLR, this is the name of the
         collection to connect to.
+    partition_len: int or None
+        The desired partition size. [default: 1024]
```

Comment:
We add a new parameter that limits the number of rows that are returned per partition.

Reply:
I looked at intake-es and it seems they use the `npartitions` as an input instead of `partition_len`. 

Let me know if you prefer that option.

Reply:
I don't have a strong preference, but consistency might be good